server/server.go

```diff
@@ -1318,8 +1318,13 @@ func (s *Server) SetReplicationModeConfig(cfg config.ReplicationModeConfig) erro
 	return nil
 }
 
-// AddLeaderCallback adds a callback in the leader campaign phase.
-func (s *Server) AddLeaderCallback(callbacks ...func(context.Context)) {
+// IsPrimary returns if the server is leader.
+func (s *Server) IsPrimary() bool {
+	return s.member.IsLeader()
+}
+
+// AddPrimaryCallback adds a callback in the leader campaign phase.
+func (s *Server) AddPrimaryCallback(callbacks ...func(context.Context)) {
 	s.leaderCallbacks = append(s.leaderCallbacks, callbacks...)
```

Comment:
The implementation is not the same as the function name.

Reply:
I will add more comments.